pandas/io/common.py

```diff
@@ -179,6 +179,8 @@ def get_filepath_or_buffer(filepath_or_buffer, encoding=None,
     filepath_or_buffer : a url, filepath (str, py.path.local or pathlib.Path),
                          or buffer
     encoding : the encoding to use to decode py3 bytes, default is 'utf-8'
+    mode : {'rb', 'wb', 'ab'} applies to S3 where a write mandates opening the
```

Comment:
mode : str, optional, you don't need to mention anything else here

Reply:
let's keep a documentation of available options, and have the comment that this applies to s3 destinations.

Reply:
no this is just distracting as it only applies to s3 and is simply a pass thru option. pls change.